### DISCUSSION TITLE
How to view results of previous query after editing and running new query

### DISCUSSION THREAD

**Discussion by laurihold78** [dbeaver/dbeaver#11039] [General]
I couldn't find an answer to my question when searching online or in the dbeaver documentation so I'm hoping someone here can help me.

Basically what happens is when I'm writing a query in the editor, once I run it the results tab pops up. This is good. But once I edit the query and re-run the code, the previous results are replaced in the results tab by the results of the updated code.

How can I keep the results of previous queries shown after running new queries in the same tab so that I can compare differences more easily? 

Thanks

**Comment by LonwoLonwo** [comment]
Duplicate of https://github.com/dbeaver/dbeaver/discussions/11008
